include/ismrmrd/xml.h

```diff
@@ -31,164 +32,157 @@
  * @{
  */
 
-namespace ISMRMRD
-{
+namespace ISMRMRD {
```

Comment:
It looks like this file was reformatted. Maybe do that in a separate PR?

Reply:
Yes, reverted that.